packages/ua-devtools/src/oapp/config.ts

```diff
@@ -435,46 +435,72 @@ export const configureReceiveConfig: OAppConfigurator = withOAppLogger(
+                continue
+            }
+            const oappSdk = await createSdk(from)
+
+            // combines enforced options together by msgType
+            const enforcedOptionsByMsgType = config.enforcedOptions.reduce(
+                enforcedOptionsReducer,
+                new Map<number, Options>()
+            )
 
-        // We ask the oapp SDK whether this config has already been applied
-        for (const [msgType, options] of enforcedOptionsByMsgType) {
-            const currentEnforcedOption: Bytes = await oappSdk.getEnforcedOptions(to.eid, msgType)
-            if (currentEnforcedOption !== options.toHex()) {
-                // Updates map with new configs for that OApp and OAppEnforcedOptionParam[]
-                const setConfigsByLibrary = setEnforcedOptionsByEndpoint.getOrElse(from, () => [])
-                setConfigsByLibrary.push({
-                    eid: to.eid,
-                    option: {
-                        msgType,
-                        options: options.toHex(),
-                    },
-                })
-                setEnforcedOptionsByEndpoint.set(from, setConfigsByLibrary)
+            // We ask the oapp SDK whether this config has already been applied
+            for (const [msgType, options] of enforcedOptionsByMsgType) {
+                const currentEnforcedOption: Bytes = await oappSdk.getEnforcedOptions(to.eid, msgType)
+                logger.verbose(
+                    `Checked current enforced options for ${formatOmniVector({ from, to })}: ${currentEnforcedOption}`
+                )
+                if (currentEnforcedOption !== options.toHex()) {
+                    // Updates map with new configs for that OApp and OAppEnforcedOptionParam[]
+                    const setConfigsByLibrary = setEnforcedOptionsByEndpoint.getOrElse(from, () => [])
+                    setConfigsByLibrary.push({
+                        eid: to.eid,
+                        option: {
+                            msgType,
+                            options: options.toHex(),
+                        },
+                    })
+                    setEnforcedOptionsByEndpoint.set(from, setConfigsByLibrary)
+
+                    const updatedEnforcedOptionsCsv = setEnforcedOptionsByEndpoint
+                        .getOrElse(from, () => [])
+                        .map(
+                            (enforcedOption) =>
+                                '{msgType: ' +
+                                enforcedOption.option?.msgType +
+                                ', options: ' +
+                                enforcedOption.option?.options +
+                                '}'
+                        )
+                        .join(', ')
+                    logger.verbose(
+                        `Set enforced options ${updatedEnforcedOptionsCsv} for ${formatOmniVector({ from, to })}`
+                    )
```

Comment:
change 4

Reply:
Hmm this may actually be redundant -- see sdk.ts [here](https://github.com/LayerZero-Labs/devtools/blob/e953380308bb575b40a55c8a7cd48469e489bc7c/packages/ua-devtools-evm/src/oapp/sdk.ts#L130-L139)

The log in this file doesn't seem to get printed, but the above log in `sdk.ts` does. Thinking to remove the logs I added here